packages/react-components/components/BackButton.v2.tsx

```diff
@@ -0,0 +1,35 @@
+import BackChevron from '@celo/react-components/icons/BackChevron.v2'
+import colors from '@celo/react-components/styles/colors.v2'
+import variables from '@celo/react-components/styles/variables'
+import * as React from 'react'
+import { StyleSheet, TouchableOpacity, TouchableOpacityProps, View } from 'react-native'
+
+type Props = TouchableOpacityProps & {
+  height?: number
+  color?: string
+  testID?: string
+}
+export default function BackButton({ onPress, color, height, disabled, testID }: Props) {
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity
+        hitSlop={variables.iconHitslop}
```

Comment:
Maybe not a problem but just a note, I think hitslop only works if the parent container is sufficiently large (ie small parent will cut off hitslop area)